mlrun/api/api/utils.py

```diff
@@ -168,8 +174,23 @@ def _generate_function_and_task_from_submit_run_body(db_session: Session, data):
         function = new_function(runtime=function_dict)
     else:
         if "://" in function_url:
-            function = import_function(
-                url=function_url, project=task.get("metadata", {}).get("project")
+            project_name = task.get("metadata", {}).get("project")
+            hub_function = function_url.startswith(mlrun.utils.hub_prefix)
```

Comment:
Can be false as it will be checked in the else clause
```suggestion
            hub_function = False
```